File: app/src/main/java/com/il2cpp/dumper/MainActivity.java
package com.il2cpp.dumper;

import android.app.Activity;
import android.os.Bundle;


public class MainActivity extends Activity {
    static {
        System.loadLibrary("il2cppdumper");
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: app/src/main/java/com/il2cpp/dumper/MainActivity.kt
package com.il2cpp.dumper

import android.app.Activity
import android.os.Bundle
import android.os.PersistableBundle

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)
        setContentView(R.layout.activity_main)
    }

    companion object {
        init {
            System.loadLibrary("native-lib")
        }
    }
}